partner_risk_insurance_security/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_full_access_user = fields.Boolean(
+        string="Is Full Access User",
+        compute="_compute_is_full_access_user",
+        store=False,
+    )
+
+    @api.depends("user_id")
+    def _compute_is_full_access_user(self):
+        for partner in self:
```

Comment:
**issue**: You're not using any field of the partner. You don't need to recompute the value of the field for every partner, you can compute it once and then assign it to all.

Even better, you can assign it all at once for every partner:

```py
self.is_full_access_user = ....
```